pkg/minikube/config/profile.go

```diff
@@ -242,6 +250,27 @@ func removeDupes(profiles []string) []string {
 	return result
 }
 
+// TODO: MachineName is copied from driver to avoid dependency cycle. Remove it or move to other module
+// MachineName returns the name of the machine, as seen by the hypervisor given the cluster and node names
+func MachineName(cc *ClusterConfig, n *Node) string {
```

Comment:
lets move this a package that everyone can use, and we could refactor all other existances of this function

Reply:
Which package is good for new home of this func?

Reply:
does it allow to put it in 'drivers"  package ?

Reply:
or can we put it in the `machine` package?
so we could call it something like this

```
machine.NameFromConfig(cc *ClusterConfig, n *Node)
```

Reply:
how about we refactor the MachineName func to not depend on clusterConfig or Node ?

so we pass everything we want from it ?

machine.Name(name string,  int nodes, controlPlane bool, ...)

Reply:
Function required following arguments,
Cluster name, Node name, number of node in cluster, is controlPlane or not
`machine.Name(clusterName, nodeName string, int nodes, controlPlane bool)`. 
Is it okay?

Reply:
sure that sounds like a good comprimise